### DISCUSSION TITLE
Welcome to AdventOfCode_WIR Discussions!

### DISCUSSION THREAD

**Discussion by CarolineMorton** [CarolineMorton/AdventOfCode_WIR#1] [Announcements]
# 👋 Welcome to Women in Rust Advent of Code Discussions!

Hello Rustaceans! Welcome to our community discussion space for the Advent of Code 2024 challenge. This is where we'll collaborate, share ideas, and support each other throughout our coding adventure! 

## 🤝 What to Use Discussions For
- Share your approaches to solving puzzles (after others have had a chance to solve them!)
- Ask questions about Rust implementation challenges
- Post helpful resources and tips
- Discuss different algorithms and optimization strategies
- Connect with other participants

## 📌 Discussion Guidelines
- No spoilers for the current day's puzzle until at least 24 hours after release
- When sharing solutions, explain your thought process
- Tag your posts appropriately (see categories below)
- Be supportive and welcoming to Rustaceans of all skill levels
- Remember that learning is more important than competition

## 🏷️ Categories
- **Q&A**: Technical questions about Rust or the puzzles
- **Solutions**: Share and discuss solutions (after the spoiler period)
- **Show and Tell**: Share interesting approaches or optimizations
- **Help Wanted**: Ask for assistance when stuck
- **General**: Any other community discussions

## 🚀 Get Started
Comment below with a quick introduction:
- Your name/handle
- Your experience with Rust
- What you're hoping to learn from Advent of Code
- Your favorite thing about Rust so far

While we'll continue using our LinkedIn group for general community chat, this is your space for sharing code examples and detailed technical discussions.

Happy coding! 🎄🦀

**Comment by Sowmiya07** [comment]
I was able to stay consistent until Day 4. I missed some problems in between. I started again today and solved Day 10 problem. 
[Here](https://github.com/Sowmiya07/AdventOfCode_WIR/blob/main/day_10/src/main.rs) is my Day 10 Part 1 solution. I will be adding more tests and part 2 solution tomorrow. Also, I will be catching up missed problems in this week.
